test/JDBC/expected/BABEL-4547-NVARCHAR.out

```diff
@@ -0,0 +1,12 @@
+
+
+DECLARE @babel4547NVarchar NVARCHAR(max) = '';
```

Comment:
Do we have enough test cases with nvarchar(max) at column level? Casting to and from nvarchar(max)?